Chapter-3-Exercises/CodeNeophyte_CH3_EX2_Recursion.js

```diff
@@ -0,0 +1,11 @@
+function isEven (number) {
+	if (number === 0 ) {
+		return true;
+	} else if (number === 1) {
+		return false;
+	} else if (number < 0) {
+		return isEven(-number);
```

Comment:
Interesting way to convert a negative number to positive to allow the function to work on negative argument!